_shows/17_18/dirk_gently.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Dirk Gently's Holistic Detective Agency
+playwright: Douglas Adams
+adptor: "James Goss & Arvind Ethan David"
+period: Spring
+season: In House
+season_sort: 3000
+date_start: 2018-05-09
+date_end: 2018-05-12
+venue: New Theatre
+
+cast:
+- role: Dirk Gently
+  name: Beth Mullen
+- role: Reg
+  name: Joe Strickland
+- role: Richard MacDuff
+  name: Daniel McVey
+- role: Gordon Way
+  name: Callum Walker
+- role: Susan Way
+  name: Margauz Valarche
```

Comment:
Should that be an x rather than a z? Also, do we support special characters (é)?